core/src/SYCL/Kokkos_SYCL_Parallel_Reduce.hpp

```diff
@@ -168,25 +168,24 @@ class ParallelReduce<FunctorType, Kokkos::RangePolicy<Traits...>, ReducerType,
     sycl::queue& q = *instance.m_queue;
 
     // FIXME_SYCL optimize
-    constexpr size_t wgroup_size = 32;
-    std::size_t size             = policy.end() - policy.begin();
-    const auto init_size =
-        std::max<std::size_t>((size + wgroup_size - 1) / wgroup_size, 1);
+    constexpr size_t wgroup_size       = 128;
+    constexpr size_t values_per_thread = 2;
```

Comment:
this values per thread, is probably right for something like dot product, but probably wrong for LAMMPS where the reduction is on rather beefy kernels which often don't have enough parallelism (think 30k atoms, each looping serially over its neighbors and then do a reduction over all of them) not sure if you care right now